.github/workflows/brakeman.yml

```diff
@@ -22,10 +22,31 @@ jobs:
 
       - name: Run Brakeman
         continue-on-error: true
-        run: bundle exec brakeman . --except CheckRenderInline --quiet -f sarif >> brakeman.sarif
+        run: |
+          if [ "${{ github.repository_visibility }}" == 'public' ]; then
```

Comment:
It might be clearer and cleaner to make these separate steps and use GitHub Actions' `if:` rather than doing it in shell.